pkg/common/config/consts_and_errors.go

```diff
@@ -66,4 +66,6 @@ var (
 
 	// ErrInvalidIPFamilyType is returned when an invalid IPFamily type is encountered
 	ErrInvalidIPFamilyType = errors.New("Invalid IP Family type")
+
+	ErrMissingDualStackEnvVar = errors.New("Two IP families provides, but dual stack feature is not enabled")
```

Comment:
This is only called in 1 place so don't think a constant is really needed.

Reply:
Yeah, that's the only place, I can get that removed.